include/orb/Orbit.h

```diff
@@ -30,20 +30,26 @@ SOFTWARE.
  */
 
 #pragma once
-#include <cstdint>
-#include <cstdlib>
-#include <cmath>
-#include <limits>
-#include <array>
-#include <lin/core.hpp>
-#include <lin/generators.hpp>
+
+#include "JacobianHelpers/jacobian_autocoded.h"
+
 #include <gnc/config.hpp>
 #include <gnc/constants.hpp>
-#include "JacobianHelpers/jacobian_autocoded.h"
+
+#include <lin/core.hpp>
+#include <lin/generators.hpp>
+#include <lin/math.hpp>
+#include <lin/queries.hpp>
 
 #include <geograv.hpp>
 #include <GGM05S.hpp>
 
+#include <array>
+#include <cstdint>
+#include <cstdlib>
+#include <cmath>
+#include <limits>
+
```

Comment:
Just some general reorganization to fit the style of the rest of the repository better.